src/dug/core/async_search.py

```diff
@@ -196,16 +195,46 @@ async def search_concepts(self, query, offset=0, size=None, fuzziness=1, prefix_
                 "minimum_should_match": 1,
             }
         }
-        body = json.dumps({'query': query})
-        total_items = await self.es.count(body=body, index="concepts_index")
+        return query_object
+
+    async def search_concepts(self, query, offset=0, size=None, types=None,
+                              **kwargs):
+        """
+        Changed to a long boolean match query to optimize search results
+        """
+        query_object = {'query': self._build_concepts_query(query, **kwargs)}
+        total_items = await self.es.count(
+            body=json.dumps(query_object),
```

Comment:
this probably should be a dict instead of json string when sending queries to es. @braswent  had some issues when using later versions of elastic search